package.json

```diff
@@ -45,6 +45,7 @@
     "anymatch": "^1.3.0",
     "async-each": "^0.1.6",
     "glob-parent": "^2.0.0",
+    "inherits": "^2.0.1",
```

Comment:
> I am using chokidar in the browser using a virtual filesystem (fs-memory)

I'm intrigued. Could you elaborate more on this?

I would expect any in-memory fs shim that provides an `fs.watch` method to work fine on its own without all the problems involved in interfacing with a real file system. What is chokidar's role in this case?